test/lit/passes/instrument-memory64.wast

```diff
@@ -406,6 +421,8 @@
   ;; CHECK-NEXT:  )
   ;; CHECK-NEXT: )
   (func $A (type $1)
+    (drop (memory.grow (i64.const 4)))
```

Comment:
Maybe add this at the end, not the start? That would make the diff far shorter.

It might be even better in a new function, for easier reading.